react-instantsearch-hooks/range-slider-with-radix-ui/src/App.js

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from 'react';
+import {
+  Hits,
+  InstantSearch,
+  Pagination,
+  RefinementList,
+  SearchBox,
+  useConnector,
+} from 'react-instantsearch-hooks-web';
+import connectRange from 'instantsearch.js/es/connectors/range/connectRange';
+import * as Slider from '@radix-ui/react-slider';
+import algoliasearch from 'algoliasearch/lite';
+
+import './App.css';
+
+const searchClient = algoliasearch(
+  'B1G2GM9NG0',
+  'aadef574be1f9252bb48d4ea09b5cfe5'
+);
+
+function App() {
+  return (
+    <div className="container">
+      <InstantSearch
+        searchClient={searchClient}
+        indexName="demo_ecommerce"
+        insights={true}
+      >
+        <div className="search-panel">
+          <div className="search-panel__filters">
+            <RefinementList attribute="brand" />
+            <RangeSlider attribute="price" />
+          </div>
+
+          <div className="search-panel__results">
+            <SearchBox className="searchbox" placeholder="Search" />
+            <Hits />
+
+            <div className="pagination">
+              <Pagination />
+            </div>
+          </div>
+        </div>
+      </InstantSearch>
+    </div>
+  );
+}
+
+function useRangeSlider(props) {
+  return useConnector(connectRange, props);
+}
+
+function RangeSlider(props) {
```

Comment:
why is it all in a single file? maybe having RangeSlider in its own file is easier for people to copy paste?

Reply:
yeah that's right, although I think the order probably could be changed, as the focus of this example is about RangeSlider, so it should be defined before the App/usage. I suggested a separate file so that can be the focused file in the codesandbox link

Reply:
Focusing the file from the docs is a fair point. I've reflected this in 5b50b594.